### PR TITLE
[17.multilingual-bot] Add License and TranslatorMiddleware in Application

### PR DESCRIPTION
**Pull request by Batta32**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java [hunk 1]
```diff
@@ -43,6 +46,13 @@ public static void main(String[] args) {
      */
     @Override
     public BotFrameworkHttpAdapter getBotFrameworkHttpAdaptor(Configuration configuration) {
-        return new AdapterWithErrorHandler(configuration);
+        Storage storage = this.getStorage();
+        ConversationState conversationState = new ConversationState(storage);
```
Comment (fran893) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java:50]: Here we can use the getConversationState from the `BotDependencyConfiguration` right? see [here](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-integration-spring/src/main/java/com/microsoft/bot/integration/spring/BotDependencyConfiguration.java#L132). Also, we can create a different method to get the conversationState, something like, `getConversationState(Configuration configuration)`

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java [hunk 2]
```diff
@@ -43,6 +46,13 @@ public static void main(String[] args) {
...
+        UserState userState = new UserState(storage);
```
Comment (fran893) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java:51]: Same as above, use the getUserState, see [here](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-integration-spring/src/main/java/com/microsoft/bot/integration/spring/BotDependencyConfiguration.java#L143). Also, we can create a different method for this, just like the conversationSate

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java [hunk 3]
```diff
@@ -43,6 +46,13 @@ public static void main(String[] args) {
      */
     @Override
     public BotFrameworkHttpAdapter getBotFrameworkHttpAdaptor(Configuration configuration) {
-        return new AdapterWithErrorHandler(configuration);
+        Storage storage = this.getStorage();
```
Comment (fran893) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java:49]: Can we create a different method to get the storage?

**Review by fran893 — CHANGES_REQUESTED** [review]

**Review by FedericoBernal — APPROVED** [review]
LGTM

**Review by fran893 — APPROVED** [review]

**Review by VictorGrycuk — APPROVED** [review]

**Pull request merged**